locales/ku/LC_MESSAGES/duckduckgo.po

```diff
@@ -3432,7 +3432,7 @@ msgstr "Na"
 
 msgctxt "noresults"
 msgid "No %s found for %s%s%s."
-msgstr "Tiştek nehate dîtin %s bo %s%s%s."
+msgstr "Tiştek nehate dîtin %sji bo %s%s%s."
```

Comment:
Ti %sek nehate dîtin ji bo %s%s%s?

Reply:
Ev format şaş bû, bnr van herdû yan belkî tu alî min bikî 
![Screenshot_2022-11-03-19-35-18-121_com vivaldi browser](https://user-images.githubusercontent.com/85353920/199780217-129f504c-e63f-4b31-bd2d-0edccd26bd37.jpg)
![Screenshot_2022-11-03-19-35-10-210_com vivaldi browser](https://user-images.githubusercontent.com/85353920/199780270-ed7b2250-4e50-4151-851b-16712ae9c2f1.jpg)

Reply:
Heke şaş nebim divê di stringa wergerkirî de wekî stringa sereke çar %s hebin,

Reply:
Tiştek ji gotarên nûçeyê bo "query" nehate dîtin

Wisa çawa ye?

Reply:
Em nikarin zindî pêşdîtin bikin ber wê bi rastî ez jê nizanim... Ka tu serê xwe pê bişîne û nerîna xwe ya dawî pêşniyar bike bo herduyan

Reply:
> Tiştek ji gotarên nûçeyê bo "query" nehate dîtin
> 
> Wisa çawa ye?

Better? I've no solution to provide honestly

Reply:
bi dilê xwe bike

Reply:
Pêşniyara xwe pêşniyar bike an update bike

Reply:
"Tiştek ji %s bi %s%s%s nehate dîtin."

Sererast ke